media/transcoder/job_template_test.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import resource_manager
+
+import create_job_template
+import delete_job_template
+import get_job_template
+import list_job_templates
+
+location = "us-central1"
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+template_id = "my-python-test-template"
+
+
+def test_template_operations(capsys):
+
+    # Enable the following APIs on the test project:
+    # *   Transcoder API
+    # *   Cloud Resource Manager API (needed for project number translation)
+
+    client = resource_manager.Client()
+    project = client.fetch_project(project_id)
+    project_number = project.number
+
+    job_template_name = (
+        f"projects/{project_number}/locations/{location}/jobTemplates/{template_id}"
```

Comment:
Most resource names use the project_id in this location, not the project_number. Is this one an exception?

Reply:
Yes, the response returns the project-number in place of the project_id.

Reply:
> the project number is the canonical identifier, and therefore APIs must use project numbers in responses, regardless of whether they were sent a project ID or a project number

https://google.aip.dev/cloud/2510